packages/lib/constants.ts

```diff
@@ -72,3 +73,18 @@ export const LOCAL_UPLOAD_URL = {
   public: new URL(`${WEBAPP_URL}/api/v1/management/storage/local`).href,
   private: new URL(`${WEBAPP_URL}/api/v1/client/storage/local`).href,
 } as const;
+
+// Enterprise License constant
+export const FORMBRICKS_ENTERPRISE_LICENSE_KEY = env.FORMBRICKS_ENTERPRISE_LICENSE_KEY;
+
+export const getIsEnterpriseEdition = () =>
+  unstable_cache(
+    async () => {
+      if (FORMBRICKS_ENTERPRISE_LICENSE_KEY) {
+        return FORMBRICKS_ENTERPRISE_LICENSE_KEY?.length > 0;
+      }
+      return false;
+    },
+    ["isEE"],
```

Comment:
please rename key to `getIsEnterpriseEdition`
(our new naming for cache keys being the name of the function)